.eslintrc.cjs

```diff
@@ -2,7 +2,10 @@ module.exports = {
   extends: [
     '@nuxtjs/eslint-config',
   ],
-  ignorePatterns: ['dist/'],
+  ignorePatterns: [
+    'dist/',
+    'templates/',
+  ],
```

Comment:
Can you also add this in typescript section so that the style matches the old JS code:

```js
        'comma-dangle': 'off',
        '@typescript-eslint/comma-dangle': ['error', 'always-multiline'],
```

?

Reply:
Done in [`96c68c0`](https://github.com/nuxt-community/sentry-module/pull/456/commits/96c68c02b608eca450021375408defd7e6a07a00)